presto-jdbc/src/test/java/io/prestosql/jdbc/TestJdbcResultSetTimezone.java

```diff
@@ -366,7 +366,8 @@ public String toString()
 
         PostgresqlReferenceDriver()
         {
-            postgresqlContainer = new PostgreSQLContainer<>("postgres:10.3");
+            // Use the current latest PostgreSQL version as the reference
+            postgresqlContainer = new PostgreSQLContainer<>("postgres:12.4");
```

Comment:
We should extract this as constant somewhere. Should we document it as well?

Reply:
> We should extract this as constant somewhere

there is no shared place for this

>  Should we document it as well?

https://github.com/prestosql/presto/pull/5058